src/main/webapp/js/instructorCourseEditAjax.es6

```diff
@@ -0,0 +1,48 @@
+/* global checkTheRoleThatApplies:false bindChangingRole:false enableEditInstructor:false */
+
+let instructorSize;
+
+$(document).ready(() => {
+    const editLinks = $('a[id^=instrEditLink]');
+    instructorSize = editLinks.length;
+    $(editLinks).click(editFormRequest);
+});
+
+function editFormRequest(e) {
+    e.preventDefault();
+    const editButton = this;
```

Comment:
Note for a future PR: this seems to be a dubious/strange sort of abstraction.

Maybe `const editButton = $(this)` followed by consistent use of `editButton` instead of `$(this)` in the rest of the function body may be better. Still, this is beyond the scope of this PR, so don't bother making the change now.